core-services/prow/02_config/_config.yaml

```diff
@@ -2145,6 +2146,16 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - redhat-developer/service-binding-operator
+  - labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    repos:
+    - redhat-developer/kam
```

Comment:
Please don't add new queries for repos with same merging criteria like repos that already have a query. This looks like it could just be added to the query above?

Reply:
Fixed.